### ISSUE TITLE
CentOS 8 install started failing with 'nothing provides python3-mock'

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/ansible-role-certbot#107]
```
TASK [geerlingguy.certbot : Install Certbot.] **********************************
fatal: [instance]: FAILED! => {"changed": false, "failures": [], "msg": "Depsolve Error occured: \n Problem: package certbot-1.3.0-3.el8.noarch requires python3-certbot = 1.3.0-3.el8, but none of the providers can be installed\n  - conflicting requests\n  - nothing provides python3-mock needed by python3-certbot-1.3.0-3.el8.noarch\n  - nothing provides python3.6dist(mock) needed by python3-certbot-1.3.0-3.el8.noarch", "rc": 1, "results": []}
```

See failed build: https://travis-ci.org/github/geerlingguy/ansible-role-certbot/jobs/672809308#L464

This started happening on cron-triggered CI jobs within the past 8 days.

**Comment by stefan0xC** [comment]
It seems you have to either wait on upstream fix (certbot/certbot#7913) or enable PowerTools repository in CentOS 8.  See also https://bugzilla.redhat.com/show_bug.cgi?id=1822959#c1

**Comment by geerlingguy** [comment]
@stefan0xC - Thanks for catching that! I hadn't had time to investigate yet.

**Comment by geerlingguy** [comment]
Fixed via above commit; this will be included in the 3.1.0 version of this role.

**Comment by ericsysmin** [comment]
@stefan0xC is there a reason you used `yum` to install `dnf-plugins-core` and not `dnf`

**Comment by geerlingguy** [comment]
@ericsysmin - `yum` aliases to `dnf` on CentOS 8. In user docs it still calls out 'yum' rather than 'dnf' in both Red Hat and CentOS docs, even though the backend is `dnf`.

**Comment by FelixSchwarz** [comment]
Just for completeness: I plan to upgrade the CentOS RPM to certbot 1.4 which also avoids this issue (and apologies for the PowerTools repo issue - I spent quite some time a few months ago but the workaround I came up with was just too ugly).

Unfortunately I need to work around another issue before certbot 1.4 will hit EPEL but I hope to do that in the coming days anyway. If you are interested, please follow https://bugzilla.redhat.com/show_bug.cgi?id=1834530
